File: program/check-even-or-odd-number/CheckEvenOrOddNumber.java
import java.util.Scanner;

public class CheckEvenOrOddNumber {
	public static void main(String[] args) throws Exception {
		System.out.print("Enter number : ");

		Scanner sc = new Scanner(System.in);

		int num = sc.nextInt();

		sc.close();

    if ( num % 2 == 0){
      System.out.println(num + " is an even number.");
    }
    else{
      System.out.println(num + " is a odd number.");
    }
    
		
	}
}

File: program/check-even-or-odd-number/check_even_or_odd_number.dart
import 'dart:io';

void main(){
  int number;
  print("Enter a number :");
  number = int.parse(stdin.readLineSync()!);
  if(number.isEven){
      print("$number is an even number");
  } else if(number.isOdd){
      print("$number is an odd number");
  }
}
